scene/gui/color_picker.cpp

```diff
@@ -599,6 +743,29 @@ void ColorPicker::erase_preset(const Color &p_color) {
 	}
 }
 
+void ColorPicker::erase_recent_preset(const Color &p_color) {
+	if (recent_presets.find(p_color)) {
```

Comment:
Assign it to a variable instead of running `find()` twice.
(also maybe it will be at the same index in `recent_preset_cache`?)

Reply:
> (also maybe it will be at the same index in `recent_preset_cache`?)

But how can we make use of that index?

Reply:
I mean you can do `int idx = recent_presets.find(p_color)`
And then `recent_presets[idx]` and probably `recent_preset_cache[idx]`. The latter assumes that `recent_presets` and `recent_preset_cache` are in sync, but I didn't check if it's true (if not, ignore it).

Reply:
> I mean you can do `int idx = recent_presets.find(p_color)`


But `find` is not returning index, it is returning the list element.